### PR TITLE
Feat: react tool map legend

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR adds the new legend built entirely with react.

### CODE REVIEW

File: frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            {[...bucket[0]].reverse().map(value => (
+              <li>{abbreviateNumber(value, 3)}</li>
+            ))}
+            {[...bucket[1]].reverse().map(value => (
+              <li>{abbreviateNumber(value, 3)}</li>
+            ))}
+          </ul>
+        )}
+
+        {!isBivariate && (
```
Comment (Bluesmile82) [inline frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx]: Ternary maybe

File: frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        )}
+        {isBivariate && (
+          <svg className="icon icon-bidimensional-legend-arrows">
+            <use xlinkHref="#icon-bidimensional-legend-arrows" />
+          </svg>
+        )}
+
+        <ul className="bullets">
+          <li>
+            <div className="bullet color-dfdfdf" style={{ background: '#DFDFDF' }} />
```
Comment (Bluesmile82) [inline frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx]: Why do we need the class and the style?

File: frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx (hunk 1 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx]: I dont love jsx ternaries, I have more difficulty reading them...

File: frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx (hunk 2 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/tool/legend/choropleth-legend.component.jsx]: good question, have no answer to it. I just translated some vanilla code. Will give it a go at cleaning this, I just didnt wanted to spend much time on this.

**Comment by sorodrigo** [comment]
I've added the missing map warnings and cleaned some of the styles. Ready to be merged

**Comment by Bluesmile82** [comment]
There is a bug here. If we change one column type the layers disappear and you are unable to select more

**Comment by sorodrigo** [comment]
You are able to select contextual layers, the choropleth layers are being set, but aren't being shown because they're only available for municipalities. The fix is to disable and remove selected layers from the state, when not in municipalities column. I managed to replicate this bug in production, and makes more sense to fix it when we add the layers modal

**Comment by Bluesmile82** [comment]
Ok, let's wait to the modal then

**Pull request merged**